packages/frontend/src/components/Explorer/FiltersCard/useFieldsWithSuggestions.ts

```diff
@@ -54,9 +60,10 @@ export const useFieldsWithSuggestions = ({
                 // converts table calculation to filterable table calculation
                 // which is a sub-type of Field.
                 const cals = (tableCalculations || []).reduce<
-                    TableCalculationField[]
+                    ItemsMap[string][]
                 >((acc, cal) => {
-                    const tableCalculationFilters: TableCalculationField = {
+                    // TODO: Verify if this is OK
+                    const tableCalculationFilters = {
```

Comment:
❓ I removed the explicit typing. what do you think?

Reply:
Can we remove the loop as we don't need to convert?

Reply:
done: https://github.com/lightdash/lightdash/pull/10062/commits/0797a01809c674faf5c70decac835d9a8e0252e8
thanks for the suggestion 🙏